README.md

```diff
@@ -1,7 +1,7 @@
 GitHub
 ------
 
-[![Build Status](https://travis-ci.org/phadej/github.svg?branch=master)](https://travis-ci.org/phadej/github)
+[![Haskell-CI](https://github.com/phadej/github/actions/workflows/haskell-ci.yml/badge.svg)](https://github.com/phadej/github/actions/workflows/haskell-ci.yml)
```

Comment:
Thanks!  The repo is now at the `haskell-github` organization.

Reply:
👍 I updated the github link in other parts of README too.